packages/shared/src/hooks/useDisableFilterAlert.ts

```diff
@@ -0,0 +1,26 @@
+import { useEffect } from 'react';
+import request from 'graphql-request';
+import { UPDATE_ALERTS } from '../graphql/alerts';
+import { apiUrl } from '../lib/config';
+import { FeedSettings } from '../graphql/feedSettings';
+import useAlertContext from './useAlertContext';
+
+export default function useDisableFilterAlert(
```

Comment:
That makes sense, though I was thinking about its requirement to `feedSettings` which will make an unnecessary request at sidebar load since the data request will only be needed from the inside of each filter.

Reply:
Now that we have a single request that fetches all the data for the sidebar that makes sense to fetch it as the sidebar opens

Reply:
Good point. Coming right up!